### PR TITLE
Add workflow figure

### LINKED ISSUE #40: Make github workflow figure
#8 has some parts about PR commenting. This would be more general. I'll start with a sketch and then we can discuss improvements.

### LINKED ISSUE #41: Identify specific deep review example
We want a contribution from reader identifies change -> file issue & discuss -> file PR & discuss -> final revisions to PR -> final merging.

@agitter do you want to come up with one of these examples that you particularly like as an exemplar workflow?

### PR DESCRIPTION
**Pull request by agitter**
Closes #40, closes #41.

Should the example issue and pull request from #41 be added to the workflow figure caption?

### CODE REVIEW

File: content/02.main-text.md
Comment (cgreene) [inline content/02.main-text.md]: I'd put the specific example here. General case described. Then specific example. Then going into the bit below about comparing unrestricted v. highly structured editing.

File: content/02.main-text.md
Comment (cgreene) [inline content/02.main-text.md]: I'd remove this and put things in with more detail below the general case.

**Comment by cgreene** [comment]
Also, do we still want to use https://github.com/greenelab/meta-review/issues/8#issuecomment-353614428 in some way to show the actual interface? It may be too confusing because that is about yet another type of discussion. Perhaps it'd be better in the manubot section describing features?

If we don't want it either place, we should probably close #8.

File: content/02.main-text.md
Comment (dhimmel) [inline content/02.main-text.md]: Can we add an HMTL comment that adds the Google Draw URL in the manuscript body.

And/or is there a way to save google draw as a file, so anyone could import it back into Google Draw? I don't see a way, so I think URL in a comment is fine:

```md
<!–– Google Drawing View Link https://docs.google.com/drawings/d/17mst0Z1RMXegeGhM6SwQSJ4nhHgMMh5csyV4MwhsgAY/edit?usp=sharing ––>
```

**Comment by dhimmel** [comment]
Reminder to update to 1.0.3:

> I bolded the "Document" and "Discussion" headers and fixed the alignment of the actions on the right hand side. Now labeled 1.0.3

**Comment by dhimmel** [comment]
> Also, do we still want to use https://github.com/greenelab/meta-review/issues/8#issuecomment-353614428 in some way to show the actual interface? It may be too confusing because that is about yet another type of discussion. Perhaps it'd be better in the manubot section describing features?

I like the following figure:

> manubot-draft

But not sure how much it fits here. Possible we could integrate panel A and B. But I think it would be more trouble than it's worth.

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md]: By put in more detail, do you mean describe the conversation in the issue and pull request?

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md]: I couldn't figure out how to link directly to the image on Google Drawing.  I added this comment in 9fe7285.

**Comment by agitter** [comment]
Now that we have such a strong workflow figure, I think the older figure in #8 is less essential.  I still like it but would be willing to skip it if we want to focus our efforts elsewhere.

However, the comment https://github.com/greenelab/meta-review/issues/8#issuecomment-334589624 reminded me that we might want to add the build process to the workflow figure.  Should we show Travis generating the update manuscript after the merge?  Or a Travis build failing and prompting the contributor to fix something?

**Comment by agitter** [comment]
e5fa68d11523ed5f9d2c7a350910b840def555f6 adds version 1.0.3 of the figure.

9f749f7dbbef78da658dfef01892380981b04528 adds a longer description of the Deep Review workflow example.  Is this what you had in mind @cgreene?  Should I use real names in addition to the generic "contributor", "maintainer", and "participant"?

**Review by cgreene — APPROVED** [review]
With the changes this LGTM 👍

**Comment by agitter** [comment]
*Commented on the wrong pull request - deleted*

**Comment by agitter** [comment]
When I build the manuscript locally, the Figure reference is broken and the caption does not show up:
[manuscript.pdf](https://github.com/greenelab/meta-review/files/2111669/manuscript.pdf)

@dhimmel any ideas?  I may not have the latest `manubot` conda environment.  There also isn't room for the figure and caption on a single page.

File: content/02.main-text.md
Comment (dhimmel) [inline content/02.main-text.md]: Let's reduce the width, since currently the text in the figure is much bigger than other text in the PDF. I wonder if we should do `width="3.5in"` or whatever inch width is most appropriate to make the font size 11 or 12.

**Review by dhimmel — COMMENTED** [review]
> When I build the manuscript locally, the Figure reference is broken and the caption does not show up

I'm not sure what the issue is. I'd suggest merging after the width is updated and we can deal with any rendering issues if they persist after the CI build's environment.

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md]: 75% width looked pretty good in one of my local builds so I changed to that in ea62ce6.  Everything else was approved so I'll merge once this build passes.

I suggest we keep #8 open until we finish discussing the GitHub review figure.

Comment (dhimmel) [reply content/02.main-text.md]: I prefer absolute width to relative width. The absolute width should be set to the width that the figure is the right size. I think absolute has some nice advantages when scaling the page IIRC.

**Pull request merged**
